README.md

```diff
@@ -0,0 +1,10 @@
+open-orchestra-libs
+===================
+
+| Service       | Badge         |
+| ------------- |:-------------:|
+| Travis | [![Build Status](https://travis-ci.org/open-orchestra/open-orchestra-libs.svg)](https://travis-ci.org/open-orchestra/open-orchestra-libs) |
+| CodeClimate (quality) | [![Code Climate](https://codeclimate.com/github/open-orchestra/open-orchestra-libs/badges/gpa.svg)](https://codeclimate.com/github/open-orchestra/open-orchestra-libs) |
+| CodeClimate (coverage) | [![Test Coverage](https://codeclimate.com/github/open-orchestra/open-orchestra-libs/badges/coverage.svg)](https://codeclimate.com/github/open-orchestra/open-orchestra-libs/coverage) |
+| Sensio Insight | [![SensioLabsInsight](https://insight.sensiolabs.com/projects/e7fb8f90-e8f0-4da4-9375-42e699038b79/big.png)](https://insight.sensiolabs.com/projects/e7fb8f90-e8f0-4da4-9375-42e699038b79) |
+| VersionEye | [![Dependency Status](https://www.versioneye.com/user/projects/551e87a7971f781c480001aa/badge.svg?style=flat)](https://www.versioneye.com/user/projects/551e87a7971f781c480001aa) |
```

Comment:
Use this one : [![Dependency Status](https://www.versioneye.com/user/projects/55967d7d6166340021000002/badge.svg?style=flat)](https://www.versioneye.com/user/projects/55967d7d6166340021000002)